### DISCUSSION TITLE
There's new Chat feature on reddit please add that!

### DISCUSSION THREAD

**Discussion by 0-BlackSpectrum-0** [Docile-Alligator/Infinity-For-Reddit#1332] [Not planned Ideas] [answered]
There's this new Chat feature on reddit which everyone uses to send DM now and to get to it I've to open it in browser and I don't recive notifications when I get the message as well since infinity doesn't yet support it so please add support for this reddit feature!
Here's link to it [Reddit Chat](https://www.reddit.com/chat)
And
**Screenshots:**

![Screenshot_20230117-134004_Chrome Canary.png](https://user-images.githubusercontent.com/79910450/212844870-ca4587f9-6ee2-4443-ab7a-2f2db67e080f.png)

![Screenshot_20230117-134029_Chrome Canary.png](https://user-images.githubusercontent.com/79910450/212844871-1ef6429b-caee-4912-ab1a-55cce4207cba.png)

**Comment by SevastianGit** [comment]
Yes please! I really want this, the messaging feature is the only reason why I still am forced to use the original reddit app.

**Comment by Wladefant** [comment, marked as answer]
Unfortunately that is not possible. Read here #430 for further information.
